lib/glific/messages.ex

```diff
@@ -244,6 +244,23 @@ defmodule Glific.Messages do
     end
   end
 
+  @doc false
+  @spec create_and_send_message_to_contacts(map(), []) :: {:ok, Message.t()}
+  def create_and_send_message_to_contacts(message, contact_ids) do
+    contact_ids
+    |> Enum.reduce(fn contact_id, _ ->
+      contact_attrs = %{
+        receiver_id: contact_id,
+        contact_id: contact_id
```

Comment:
I think you don't need to set a contact ID here. It will be set by create_message funcation.

Reply:
Yes, will remove it.